core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageConfigBuildStep.java

```diff
@@ -91,33 +84,14 @@ void build(SslContextConfigurationRecorder sslContextConfigurationRecorder,
 
             if (graalVmHome != null) {
                 Path graalVmLibDirectory = Paths.get(graalVmHome, "jre", "lib");
-                Path linuxLibDirectory = graalVmLibDirectory.resolve("amd64");
-                Path linuxPath = linuxLibDirectory.resolve(LIB_SUN_EC);
 
                 // We add . as it might be useful in a containerized world
                 javaLibraryPathAdditionalPath.produce(new JavaLibraryPathAdditionalPathBuildItem("."));
```

Comment:
@gsmet Do we want to keep this? I'd answer no but I'm not sure if it was added for the SunEC lib or some other reason.

Reply:
I think we can remove it.